qemu/tests/cfg/enable_scatter_windows.cfg

```diff
@@ -10,13 +10,14 @@
     driver_verifier = netkvm
     Win2016, Win2019, Win8..1, Win2012..r2:
         driver_verifier += " ndis"
-    installed_path = "C:\Program Files\Wireshark\tshark.exe"
-    check_installed_cmd = 'dir "${installed_path}"|findstr /I tshark.exe'
+    WinDump_name = "WinDump.exe"
+    installed_path = "C:\WinDump.exe"
+    check_installed_cmd = 'dir "${installed_path}" | findstr /I "${WinDump_name}"'
     x86_64:
         wireshark_name = "Wireshark-win64-1.10.1.exe"
     i386, i686:
         wireshark_name = "Wireshark-win32-1.10.1.exe"
-    install_wireshark_cmd = "xcopy WIN_UTILS:\${wireshark_name} c:\ /y && c:\${wireshark_name} /S"
+    installation_cmd = "xcopy WIN_UTILS:\${WinDump_name} c:\ /y  && xcopy WIN_UTILS:\${wireshark_name} c:\ /y && c:\${wireshark_name} /S"
```

Comment:
Seems this cmd will copy windump and wireshark to C and then install wireshark, is this expected?

Reply:
Yes

Reply:
As the title of this PR says: "Netkvm: Replace tshark.exe with WinDump.exe".
Why does your code still install wireshark but not windump?

Reply:
At the end of this case, we still need to analyze the packet's result with Wireshark only. However, we have used Windump to capture NetFlow data since Wireshark has a bug.

Reply:
For the WinDump part,  WinDump.exe is a portable software that does not need to be installed.

Thank @leidwang and let me think about the patch title again : )